src/systems/ec/implementations/react/packages/button/Button.jsx

```diff
@@ -32,18 +24,18 @@ const Button = ({
 
 Button.propTypes = {
   variant: PropTypes.string,
-  typeAttribute: PropTypes.string,
+  type: PropTypes.string,
   label: PropTypes.string,
-  extraClasses: PropTypes.string,
-  extraAttributes: PropTypes.string,
+  icon: Icon,
```

Comment:
This is not a valid propType. 

> Warning: Button: type specification of prop `icon` is invalid; the type checker function must return `null` or an `Error` but returned a object. You may have forgotten to pass an argument to the type checker creator (arrayOf, instanceOf, objectOf, oneOf, oneOfType, and shape all require an argument).

`PropTypes.shape(Icon.propTypes)` should to the trick

Reply:
done